github_issue_summarization/notebooks/requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy
```

Comment:
if you don't mind please use pipenv

Reply:
Wrapping a seldon-core microservice requires a `requirements.txt` file: https://github.com/SeldonIO/seldon-core/blob/master/docs/wrappers/python.md#create-a-model-folder